### ISSUE TITLE
Find All References low contrast

### ISSUE THREAD

**Issue by rebornix** [Monokai/monokai-pro-vscode#21]
Use the default one:

![image](https://user-images.githubusercontent.com/876920/30251749-ed53fa8e-961a-11e7-83a5-d3d451c35f2d.png)

**Comment by talbet** [comment]
Until this is updated in the package, you can add this to your user settings to override the theme and fix temporarily

```json
    "workbench.colorCustomizations": {
        "peekViewResult.fileForeground": "#ccc"
    }
```

**Comment by Monokai** [comment]
Fixed in 1.1.2
